client/multisite-dashboard/components/multisite-dashboard/multisite-dashboard.scss

```diff
@@ -0,0 +1,3 @@
+body.is-group-multisite-dashboard {
```

Comment:
What do you think of giving a shot at some other styling libraries? I mean experimenting with new technologies here, or use something that's better than SCSS. Some CSS-in-JS library or Tailwind, for example? Do we have initiatives inside the company?

cc @vindl @Automattic/team-calypso-frameworks

Reply:
It doesn't look too difficult... We could narrow the sources to this module exclusively so we could prototype.

The thing is: are we willing to prototype here? How comfortable are we about switching back to SCSS if people don't like Tailwind?

https://tailwindcss.com/docs/configuration

Reply:
I would like to try Tailwind out on a project, but I'm not sure this is the right one. We're going to be reusing mostly `@automattic/components` and `@wordpress/components` rather than starting from scratch, but since they're not using Tailwind the only Tailwind we'd be writing would be the padding rules. Tailwind does let you set the `big-padding` and `small-padding` type rules to be whatever you want so it'd give us a tiny amount of benefit, but (1) we can just use CSS or SCSS variables for that, and (2) I don't want to find ourselves in the position where our team ends up creating the Tailwind design system that all of Calypso uses (kinda like @SaxonF mentioned today on the call, about "big padding" always meaning 24px for all of our projects).

If we were defining our own components then it might be cool to try it out. Then I think we'd really be able to put it through it's paces and see how it feels. But it doesn't seem worth it for what we'll require in this project.

I think there's some CSS-in-JS usage already in Calypso. Emotion maybe? Maybe also overkill in this situation 😅 I don't think many of our CSS will need to change based on props for instance, but at least we know it's already in the Calypso bundle and so not adding more stuff to the project.

Reply:
Yep it's emotion: https://github.com/Automattic/wp-calypso/blob/f3e94eff661543b56758f23588ae1ecb6fb979f4/client/components/eligibility-warnings/card/index.tsx#L2

Reply:
I'd personally love to see experiments with Tailwind 😉 It's pretty powerful and could potentially help resolve some of our styling overlap and loading order problems. For an app of Calypso's magnitude, it could cause its own problems, of course, not to mention the cost of a potential migration 😉 In any case, though, since we're ideally leaning towards using `@wordpress/components` as our base component library, I'd recommend any experiments with Tailwind to happen there first. Same applies to CSS-in-JS: we already use `emotion` both here and there.

Reply:
Yep I think we need to wait until discussions like this pblw7J-ZM-p2 are resolved before making any big changes to our styling framework. That being said, it doesn't mean we can't start to formalise design tokens like spacing if they don't already exist.

Our scale should look something like:
<img width="1382" alt="image" src="https://user-images.githubusercontent.com/1072756/176339949-6845c43e-582c-45e5-b944-7d23a56c9332.png">